c/tskit/core.h

```diff
@@ -96,7 +96,7 @@ to the API or ABI are introduced, i.e., internal refactors of bugfixes.
 #define TSK_FILE_FORMAT_NAME          "tskit.trees"
 #define TSK_FILE_FORMAT_NAME_LENGTH   11
 #define TSK_FILE_FORMAT_VERSION_MAJOR 12
-#define TSK_FILE_FORMAT_VERSION_MINOR 1
+#define TSK_FILE_FORMAT_VERSION_MINOR 2
```

Comment:
Probably not worth bothering with the minor version bump here? We've not released it so I think it's reasonable to merge together all the compatible changes between releases into one minor version bump. Doesn't seem much point in having lots of transient file versions.

Reply:
Cool, as long one PR that needs to in a release increments should be good. I wasn't sure how many users we had who are on the bleeding edge.

Reply:
Trying to discourage bleeding edge users, but this is harmless anyway since the changes are backward/forward compatible.